### PR TITLE
[EDU-3148] refactor: modify main settings for secure menu

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Modified main settings to create simple HTTP application

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-3148]: https://aziontech.atlassian.net/browse/EDU-3148?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 1]
```diff
@@ -51,32 +49,18 @@ Choose between the supported [delivery ports](/en/documentation/products/edge-ap
 
 1. Go to the **Main Settings** tab.
 2. Under **Ports HTTP**, in addition to port `80`, select port `8008`.
-3. Under **Ports HTTPS**, in addition to port `443`, select port `8443`.
 4. Click the **Save** button.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:52]:
```suggestion
3. Click the **Save** button.
```

File: src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 2]
```diff
@@ -51,32 +49,18 @@ Choose between the supported [delivery ports](/en/documentation/products/edge-ap
...
 
-Wait a few minutes for the changes to propagate to the edge and try to access your application using one of the new ports; for example, navigate to `https://xxxxxxxxxx.map.azionedge.net:8443` and check if your application is running on the custom port.
+Wait a few minutes for the changes to propagate to the edge and try to access your application using one of the new ports; for example, navigate to `http://xxxxxxxxxx.map.azionedge.net:8008` and check if your application is running on the custom port.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:54]:
```suggestion
Wait a few minutes for the changes to propagate to the edge and try to access your application using one of the new ports. For example, navigate to `http://xxxxxxxxxx.map.azionedge.net:8008` and check if your application is running on the custom port.
```

**Review by MarianaAguilera — APPROVED** [review]
left a couple of suggestions.

File: src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 3]
```diff
@@ -28,21 +28,19 @@ The following steps will guide you through editing an edge application's setting
...
 
 1. Go to the **Main Settings** tab.
-2. Under **Delivery Protocol**, switch to **HTTP & HTTPS**.
+2. Under **Delivery Protocol**, select **HTTP**.
 3. Click the **Save** button.
 
-After some propagation time, you'll be able to access your edge application using the HTTPS protocol. To test it, access the `https://xxxxxxxxxx.map.azionedge.net` address. You should still see the configured application, only now the request was made using a more secure protocol.
-
-You can also make the following cURL request in your terminal with the HTTPS schema:
+After some propagation time, you'll be able to access your edge application using the HTTP protocol. To test it, access the `http://xxxxxxxxxx.map.azionedge.net` address. You can also make the following cURL request in your terminal with the HTTP schema:
```
Comment (hannah-sk) [inline src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:34]:
```suggestion
After the propagation time, you'll be able to access your edge application using the HTTP protocol. To test it, access the `http://xxxxxxxxxx.map.azionedge.net` address. You can also make the following cURL request in your terminal with the HTTP schema:
```

**Review by hannah-sk — APPROVED** [review]
deixei uma sugestão e de acordo com as da @MarianaAguilera ✌️

File: src/content/docs/pt-br/pages/build-jornada/edite-edge-app/main-settings.mdx [hunk 4]
```diff
@@ -131,11 +114,8 @@ curl --location --request PATCH 'https://api.azionapi.net/edge_applications/<edg
...
-  | `http_port` | Define portas HTTPS para a aplicação. Neste caso, o array `[80, 8008]` define que ambas as portas 80 e 8008 podem ser usadas por solicitações HTTP. |
-  | `https_port` | Define portas HTTPS para a aplicação. Neste caso, o array `[443, 8443]` define que ambas as portas 443 e 8443 podem ser usadas por solicitações HTTP. |
-  | `minimum_tls_version` | Quando definido como `tls_1_2`, estabelece que a versão mínima do TLS deve ser 1.2. |
-  | `supported_ciphers` | Define o conjunto de cifras suportado, sendo seu valor neste caso `TLSv1.2_2021`. Veja a lista completa de [lista de cifras suportadas](/pt-br/documentacao/produtos/edge-application/main-settings/#cifras) para mais detalhes.
+  | `delivery_protocol` | Quando definido como `http`, habilita o suporte ao protocolo HTTP. |
+  | `http_port` | Define portas HTTP para a aplicação. Neste caso, o array `[80, 8008]` define que ambas as portas 80 e 8008 podem ser usadas por solicitações HTTP. |
   | `edge_functions` | Habilita o módulo **Edge Functions**. |
   | `application_acceleration` | Habilita o módulo **Application Acceleration**. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/edite-edge-app/main-settings.mdx:120]:
```suggestion
  | `delivery_protocol` | Quando definido como `http`, habilita o suporte ao protocolo HTTP |
  | `http_port` | Define portas HTTP para a aplicação. Neste caso, o array `[80, 8008]` define que ambas as portas 80 e 8008 podem ser usadas por solicitações HTTP |
  | `edge_functions` | Habilita o módulo **Edge Functions** |
  | `application_acceleration` | Habilita o módulo **Application Acceleration** |
```

**Review by MarianaAguilera — APPROVED** [review]

**Review by hannah-sk — APPROVED** [review]

**Pull request merged**